DapperLearning/Queries.cs

```diff
@@ -2,33 +2,34 @@
 using DapperLearning.Constants;
 using System;
 using System.Data.SqlClient;
+using System.Threading.Tasks;
 
 namespace DapperLearning
 {
     internal static class Queries
     {
-        public static void GetUsers(SqlConnection connection)
+        public async static Task GetUsers(SqlConnection connection)
```

Comment:
Now because we made this methos async, it's good practice to also rename the methods to be post fixed with async: `GetUsersAsync`